common-util/src/main/java/org/sunbird/common/responsecode/ResponseMessage.java

```diff
@@ -319,6 +319,8 @@
         "Cassandra connection establishment failed in {0} mode.";
     String COMMON_ATTRIBUTE_MISMATCH = "{0} mismatch of {1} and {2}";
     String MULTIPLE_COURSES_FOR_BATCH = "A batch cannot belong to multiple courses.";
+    String SYSTEM_ALREADY_INITIALISED = "System already initialised.Cannot initialise again.";
+    String REQUEST_NOT_ALLOWED = "Request from external addresses are not allowed";
```

Comment:
Change message:
RESTRICTED_REQUEST = "Request is not allowed from current IP address."